src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/VideoFanTakeover.java

```diff
@@ -100,24 +104,24 @@ public void clickOnAdImage(){
   }
 
   public void clickOnVideoCloseButton() {
-    wait.forElementVisible(By.cssSelector(String.format(UI_ELEMENT_CLOSE_BUTTON_FORMAT, slotName))).click();
+    scrollToAdsElement(videoCloseButton(slotName), 60);
+
+    videoCloseButton(slotName).click();
     PageObjectLogging.log("clickOnVideoCloseButton", "close video button clicked", true, driver);
   }
 
   public Double getCurrentVideoTimeOnDesktop() {
```

Comment:
consider changing `Desktop` to `Wiki` because it's applicable for both mobile and desktop